app/routes/models/application-list.js

```diff
@@ -5,13 +5,11 @@ const getStyleClassByStatus = require('../../constants/status')
 const keys = require('../../session/keys')
 const { serviceUri } = require('../../config')
 
-class ViewModel {
-  constructor (request, page) {
-    return (async () => {
-      this.model = await createModel(request, page)
-      return this
-    })()
-  }
+const viewModel = (request, page) => {
+  return (async () => {
+    this.model = await createModel(request, page)
+    return this
```

Comment:
In every other app we use ViewModel not viewModel, I thought the pattern was that because it was a class name you capitalise the first letter